Google Scholar.js

```diff
@@ -9,61 +9,239 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2017-09-24 01:38:44"
+	"lastUpdated": "2017-10-07 15:13:24"
 }
 
-/*
- * Test pages
- *
- * Searches of Google Scholar with the following terms should yield a folder
- * icon that works.  Check that unlinked ([CITATION]) items that provide
- * no BibTeX data (there is currently one under "Marbury v. Madison",
- * and "clifford" seems to be a good source of garbage) are
- * dropped from the listings:
- *
- *   marbury v madison
- *   kelo
- *   smith
- *   view of the cathedral
- *   clifford
- *
- * "How cited" pages should NOT yield a page or folder icon.  The
- * Urls to these currently look like this:
- *
- *   http://scholar.google.co.jp/scholar_case?about=1101424605047973909&q=kelo&hl=en&as_sdt=2002
- *
- * Case pages should present a document icon that works:
- *
- *   http://scholar.google.co.jp/scholar_case?case=18273389148555376997&hl=en&as_sdt=2002&kqfp=13204897074208725174&kql=186&kqpfp=16170611681001262513#kq
- */
 
-var bogusItemID = 1;
+
+// attr()/text() v2
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+
 
 var detectWeb = function (doc, url) {
-	// Icon shows only for search results and law cases
+	/* Detection for law cases, but not "How cited" pages,
+	 * e.g. url of "how cited" page:
+	 *   http://scholar.google.co.jp/scholar_case?about=1101424605047973909&q=kelo&hl=en&as_sdt=2002
+	 */
 	if (url.indexOf('/scholar_case?') != -1
 		&& url.indexOf('about=') == -1
 	) {
-			return "case";
-	} else if(url.indexOf('/citations?') != -1) {
+		return "case";
+	} else if (url.indexOf('/citations?') != -1) {
+		if (getProfileResults(doc, true)) {
+			return "multiple";
+		}
+		
 		//individual saved citation
-		var link = ZU.xpathText(doc, '//a[@class="gsc_title_link"]/@href');
+		var link = ZU.xpathText(doc, '//a[@class="gsc_vcd_title_link"]/@href');
 		if(!link) return;
 		
-		if(link.indexOf('/patents?') != -1) {
-			return 'patent';
-		} else if(link.indexOf('/scholar_case?') != -1) {
+		
+		if (link.indexOf('/patents?') != -1) {
+			return 'patent';// TODO no example found
```

Comment:
I think that there are no examples for single page about patent in GoogleScholar and because they are now all on Google directly, e.g. https://www.google.com/patents/US20160328455, I am quite certain we don't need this case anymore. If a patent occurs among the search results in Google Scholar, then we will still be able to handle that.